crates/matrix-sdk-indexeddb/src/crypto_store/mod.rs

```diff
@@ -1110,6 +1182,30 @@ struct GossipRequestIndexedDbObject {
     unsent: bool,
 }
 
+/// The objects we store in the inbound_group_sessions2 indexeddb object store
+#[derive(Debug, serde::Serialize, serde::Deserialize)]
+struct InboundGroupSessionIndexedDbObject {
+    /// (Possibly encrypted) serialisation of a
+    /// [`matrix_sdk_crypto::olm::group_sessions::PickledInboundGroupSession`]
+    /// structure.
+    data: Vec<u8>,
```

Comment:
This name is bleh, but `serialized_pickled_inbound_group_session` is a mouthful...

Reply:
well we could go with `encrypted_serialized_picked_inbound_group_session` :innocent: 

Have renamed to `pickled_session` which seems like a reasonable compromise to me.